ifacemonitor/update_filter.go

```diff
@@ -98,19 +100,29 @@ mainLoop:
 					ReadyAt: u.Time.Now().Add(delay),
 					Update:  linkUpd,
 				})
-		case addrUpd := <-addrInC:
-			idx := addrUpd.LinkIndex
+		case routeUpd := <-routeInC:
+			logrus.WithField("route", spew.Sdump(routeUpd)).Debug("Route update")
```

Comment:
Curious what Sdump output looks like.  Does it cross multiple lines, and if so does that matter and look OK in the log output?

Reply:
Didn't intend to leave that in, it'll calculate the string every time even if debug is disabled.